modules/gdnative/include/gdnative/gdnative.h

```diff
@@ -136,6 +136,9 @@ typedef float godot_real;
 /////// Object (forward declared)
 typedef void godot_object;
 
+/////// naked pointer (forward declared)
+typedef void godot_pointer;
```

Comment:
Shouldn't this be `void *`?

Reply:
Nope, this replicates the "gogot_object" defined above it, it defines the container type itself.